mlir/include/mlir/Dialect/OpenMP/OpenMPClauses.td

```diff
@@ -629,12 +617,10 @@ class OpenMP_InReductionClauseSkip<
 
   let extraClassDeclaration = [{
     /// Returns the reduction variables.
-    SmallVector<Value> getReductionVars() {
+    SmallVector<Value> getAllReductionVars() {
```

Comment:
This only returns the `in_reduction` vars. I assume this is because this is for the `in_reduction` clause. Does it get merged with reductions from other clauses somewhere?